Add Documentation for userContentBaseUrl Configuration in Upptime configuration.md (#53)
upptime/upptime.js.org: docs/configuration.md

  apiBaseUrl: https://api.github.com
```

#### Custom raw content URL 

> By default, Upptime uses https://raw.githubusercontent.com to fetch raw content from GitHub repositories. This document provides instructions on how to set a custom userContentBaseUrl if you are using a proxy or another content delivery URL.

Reviewer: ```suggestion
By default, Upptime uses https://raw.githubusercontent.com to fetch raw content from GitHub repositories. If you are using a proxy or another content delivery URL, you can replace the default user content base URL.
```
